dojo/tools/openvas/csv_parser.py

```diff
@@ -71,6 +72,31 @@ def map_column_value(self, finding, column_value):
             finding.unsaved_endpoints[0].port = int(column_value)
 
 
+class CveColumnMappingStrategy(ColumnMappingStrategy):
+    def __init__(self):
+        self.mapped_column = "cves"
+        super(CveColumnMappingStrategy, self).__init__()
+
+    def map_column_value(self, finding, column_value):
+        if "," in column_value:
+            finding.description += "\n**All CVEs:** " + str(column_value)
+            finding.unsaved_vulnerability_ids.append(column_value.split(",")[0])
```

Comment:
If there are multiple CVEs in this list, we should probably save more than just the first one